etc/profile-a-l/bcompare.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for Beyond Compare by Scooter Software
+# Description: directory and file compare utility
+# Disables the network, which only impacts checking for updates.
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include bcompare.local
+# Persistent global definitions
+include globals.local
+
+# In case the user decides to include disable-programs.inc, still allow KDE's Gwenview to view images via right click -> Open With -> Associated Application
+noblacklist ${HOME}/.config/gwenviewrc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+# Allow comparing e.g. Firefox profile files, so don't disable access to programs
+# include disable-programs.inc
+# Uncommenting this breaks launch
+# include disable-shell.inc
+include disable-write-mnt.inc
+# Don't disable ${DOCUMENTS}, ${MUSIC}, ${PICTURES}, ${VIDEOS}
+# include disable-xdg.inc
```

Comment:
```suggestion
```

Reply:
I've kept such lines because to me it was surprising to realize that `xdg` was related to `${DOCUMENTS}`, and I thought others who base the profile on this one might benefit from the comment. For the same reason, I added potentially superfluous comments like

```
# Might break Pulse Audio
#machine-id
```

or

```
# Allow associated applications to play sound files
#nosound
```